### PR TITLE
DESK-1256 DESK-1254 DESK-1194 DESK-1191 DESK-1183 DESK-1257 activate terminal

### PR DESCRIPTION
**Pull request by JamieRuderman**
### Description

- Bring window to front and development on auto-launching
- DESK-1262 fixed share email field display
- Fix device name display in in-app notices
- Fix handling for non standard licensing plans
- DESK-1194 implement max connection name length
- DESK-1191 remove all AWS specific logic
- DESK-1183 DESK-1257 go to active device list when registering via code + fix register textfield size

### Ticket

<https://remoteit.atlassian.net/browse/DESK-1256>
<https://remoteit.atlassian.net/browse/DESK-1254>
<https://remoteit.atlassian.net/browse/DESK-1194>
<https://remoteit.atlassian.net/browse/DESK-1191>
<https://remoteit.atlassian.net/browse/DESK-1183>
<https://remoteit.atlassian.net/browse/DESK-1257>

### CODE REVIEW

File: frontend/src/components/LaunchSelect/LaunchSelect.tsx [hunk 1]
```diff
@@ -67,13 +69,17 @@ export const LaunchSelect: React.FC<Props> = ({ service, connection }) => {
...
-          <List disablePadding>
+          <List disablePadding className={css.indent}>
             <InlineTemplateSetting
               connection={connection}
               service={service}
               context={app.launchType === 'COMMAND' ? 'copy' : 'launch'}
             />
             <CustomAttributeSettings connection={connection} service={service} />
+            <TestUI>
+              {' '}
```
Comment (mauricio0000) [inline frontend/src/components/LaunchSelect/LaunchSelect.tsx:80]: do you need thi empty space?

File: frontend/src/pages/OrganizationAddPage.tsx [hunk 2]
```diff
@@ -60,10 +60,12 @@ export const OrganizationAddPage = () => {
           <Typography variant="h1">
             <Title>Add Organization Members</Title>
           </Typography>
-          <ContactSelector
-            contacts={contacts.filter(c => !members.find(s => s.user.id === c.id))}
-            onChange={setEmails}
-          />
+          <Gutters top={null}>
+            <ContactSelector
+              contacts={contacts.filter(c => !members.find(s => s.user.id === c.id))}
```
Comment (mauricio0000) [inline frontend/src/pages/OrganizationAddPage.tsx:65]: maybe move the filter before of the return. what do you think?

File: frontend/src/components/DialogApp/DialogApp.tsx [hunk 3]
```diff
@@ -48,24 +56,30 @@ export const DialogApp: React.FC<{
...
     <>
       <Dialog open={openApp} onClose={closeAll} maxWidth="sm" fullWidth>
-        <DialogTitle >{title}</DialogTitle>
+        <DialogTitle>{title}</DialogTitle>
         <DialogContent>
           <DialogContentText>
-            You can use a browser-based {type} launcher to access this connection.
-            Alternatively, you can download and install {type}
+            You can use a browser-based {type} launcher to access this connection. Alternatively, you can download and
+            install PuTTy
```
Comment (mauricio0000) [inline frontend/src/components/DialogApp/DialogApp.tsx:63]: I was using `type` becouse there are more options than Putty

Comment (JamieRuderman) [reply frontend/src/components/DialogApp/DialogApp.tsx:63]: Ah this was a merge conflict- not intentional - good catch

File: frontend/src/pages/OrganizationAddPage.tsx (hunk 2 above)
Comment (JamieRuderman) [reply frontend/src/pages/OrganizationAddPage.tsx:65]: Sounds good!

File: frontend/src/components/LaunchSelect/LaunchSelect.tsx (hunk 1 above)
Comment (JamieRuderman) [reply frontend/src/components/LaunchSelect/LaunchSelect.tsx:80]: I do not - I think the code formatter adds those sometimes!

**Review by mauricio0000 — APPROVED** [review]

**Pull request merged**